packages/vite/src/index.ts

```diff
@@ -128,7 +128,7 @@ export function imagetools(userOptions: Partial<VitePluginOptions> = {}): Plugin
         if (viteConfig.command === 'serve') {
           const id = await generateImageID(srcURL, config, img)
           generatedImages.set(id, image)
-          metadata.src = basePath + id
+          metadata.src = path.join(viteConfig?.server?.origin ?? '', basePath) + id;
```

Comment:
I don't think this will work on Windows and needs to be:
```suggestion
          metadata.src = path.posix.join(viteConfig?.server?.origin ?? '', basePath) + id;
```

Reply:
Ah that makes sense!